docs/Android/privacy-config.md

```diff
@@ -190,6 +205,9 @@ This is an example combination of both Disabling Attributes and Disabling Events
     "NetworkData": {
       "AttributesToDisable": [],
       "EventsToDisable": ["network_change"]
+    },
+    "AppUsageData": {
+      "DisableAll": false
     }
   } 
 }
```

Comment:
Do you mind updating the links for `Battery`, `Device` and `Network` for both Android and iOS below?